pulp-glue/pulp_glue/ansible/context.py

```diff
@@ -32,20 +33,23 @@ class PulpAnsibleRoleContext(PulpContentContext):
     ENTITIES = _("ansible roles")
     HREF = "ansible_role_href"
     ID_PREFIX = "content_ansible_roles"
+    NEEDS_PLUGINS = [PluginRequirement("ansible", min="0.7.0")]
 
 
 class PulpAnsibleCollectionVersionSignatureContext(PulpContentContext):
     ENTITY = _("ansible collection version signature")
     ENTITIES = _("ansible collection version signatures")
     HREF = _("ansible_collection_version_signature_href")
     ID_PREFIX = "content_ansible_collection_signatures"
+    NEEDS_PLUGINS = [PluginRequirement("ansible", min="0.7.0")]
```

Comment:
Is this true? Since when do we have CollectionSignatures?

Reply:
They were added in 0.12, upload support was added in 0.13.